README.md

```diff
@@ -31,11 +31,13 @@ See also: https://github.com/vbauer/java-blogs
 * Cult of Android http://www.cultofandroid.com
 * Droid Life http://www.droid-life.com
 * FXP http://fxpblog.co
+* Hootsuite http://code.hootsuite.com/category/code/android/
 * Novoda http://novoda.com/blog
 * Phandroid http://phandroid.com
 * Planet Android http://www.planetandroid.com
 * Talk Android http://www.talkandroid.com
 * Team Android http://www.teamandroid.com
+* WeddingParty http://nerds.weddingpartyapp.com/tech/categories/android/
```

Comment:
That's really weird, although wedding party is a company
https://www.weddingpartyapp.com/ don't why they didn't create an
organization on GitHub
On 11-Aug-2015 1:51 am, "Vladislav Bauer" <notifications@github.com> wrote:

> In README.md
> <https://github.com/vbauer/android-blogs/pull/9#discussion_r36679343>:
>
> >  * Novoda http://novoda.com/blog
> >  * Phandroid http://phandroid.com
> >  * Planet Android http://www.planetandroid.com
> >  * Talk Android http://www.talkandroid.com
> >  * Team Android http://www.teamandroid.com
> > +* WeddingParty http://nerds.weddingpartyapp.com/tech/categories/android/
>
> Hmm.. I see on blog a link to the personal Github account (instead of
> organization account).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vbauer/android-blogs/pull/9/files#r36679343>.
>